src/main/java/ReportGenerator.java

```diff
@@ -0,0 +1,68 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+public class ReportGenerator {
+	ArrayList<Student> students;
```

Comment:
it's better to use interfaces as much as possible. In this case, it should be List, and concrete class should be ArrayList. This way your code will be more 'flexible'